libhoney.go

```diff
@@ -204,7 +197,6 @@ func Init(config Config) error {
 	blockOnResponses = config.BlockOnResponse
 
 	// reset the global transmission
-	tx.Stop()
```

Comment:
ah was this the change I heard being discussed where it might leak if called twice?  if so, no biggie.